### ISSUE TITLE
Why do you use features from object detector for building feature bank for Nouns in Epic Kitchen?

### ISSUE THREAD

**Issue by junting** [facebookresearch/video-long-term-feature-banks#58]
Hi,

Thanks for sharing your code and models!  They just work perfectly!  I notice that for epic kitchen different from the other two datasets, you used feature extract using an object detector (instead of a 3D-CNN baseline). And what is the reason behind this?  And furthermore, I would like to know why position encodings were never used in your feature banks? 

Thanks in advance for your reply!

**Comment by chaoyuaw** [comment]
Hi @junting , 

Thanks for using LFB!

We use object features for EPIC Kitchens because that's a task that requires finding objects. Sec 5.1 of our paper provides more explanation. 

We tried positional encodings, but didn't find them to be helpful. Maybe knowing the order isn't helpful for the tasks we run on.

**Comment by junting** [comment]
Hi @chaoyuaw ,

Thanks for your quick response! :) I agree with you that object features indeed would help in the Noun task. But, I think the 3D-CNN baseline for Noun classification also can be used to extract such object features, so I am curious why this setting was not considered in your experiments.

Thanks!

**Comment by chaoyuaw** [comment]
Hi @junting, Thanks for your comment. I think what you propose is a good experiment which worths trying, too :) There wasn't a particular reason other than that we don't have space/time to try and report too many things in paper 😉

**Comment by junting** [comment]
Hi @chaoyuaw,  totally understandable! Thanks again for your reply!

**Comment by junting** [comment]
Hi @chaoyuaw ,

I can only see that the object features form the Validation and Train set for EPIC Kitchens, I wonder is it possible for you to also provide the features for the Test set?

Thanks!

**Comment by chaoyuaw** [comment]
Hi @junting , Unfortunately I don't think it's publicly available, though I should have released it... sorry. And unfortunately I also don't have access to the files I used before other than what's released. 

But we do release the model weight that can be used to extract object features. There's some instructions at https://github.com/facebookresearch/video-long-term-feature-banks/blob/master/GETTING_STARTED.md#epic-kitchens-noun-lfb
I hope that would of some help.

**Comment by junting** [comment]
Hi @chaoyuaw, Thanks for your reply! I was able to extract the object feature with the pre-trained weights that you provided.
